src/styles/partials/components/_search.scss

```diff
@@ -0,0 +1,38 @@
+.dg_search-container {
+  position: relative;
+  width: 100%;
+  height: $base-spacing-unit * 10;
+  color: $black;
+
+  .dg_search-btn {
+    display: block;
+    position: absolute;
+    right: $base-spacing-unit * 2;
+    top: $padding-small;
+    font-size: $base-font-size -2;
```

Comment:
font-size is broken in the browser, it doesn't understand `$base-font-size -2`. I think it looks ok without this being specified though.

Reply:
removed the subtraction